python/src/aiconfig/schema.py

```diff
@@ -59,9 +59,25 @@ class PromptMetadata(BaseModel):
     class Config:
         extra = "allow"
 
+class Attachment(BaseModel):
+    '''
+    Attachment used to pass data in PromptInput for non-text inputs (ex: image, audio)
+    '''
+    # index of the attachment
+    input_count: Union[int, None] = None
```

Comment:
Is this needed if attachments is a list below?

Reply:
Yea you're probably right, will remove